### PR TITLE
Move HOME to /tmpfs.

### LINKED ISSUE #1670: Kokoro disk is full
While trying to investigate what I've broken in https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1651 I stumbled upon "disk full" messages like:
```
sqlite3.OperationalError: database or disk is full
```
```
capnp.lib.capnp.KjException: kj/io.c++:406: overloaded: ::writev(fd, current, iov.end() - current): No space left on device; fd = 10
```
Found these in "Xilinx Series 7 - Vendor Tool Tests (Presubmit)" logs.

### PR DESCRIPTION
**Pull request by litghost**
Fixes #1670

### CODE REVIEW

File: .github/kokoro/steps/movehome.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+echo
+echo "========================================"
+echo "Moving home directory."
+echo "----------------------------------------"
+echo
+echo "Old home directory: ${HOME}"
+echo
+mkdir home
+cp -r $HOME/. home/
```
Comment (mithro) [inline .github/kokoro/steps/movehome.sh:11]: Why not `mv` and symlink?
```shell
mv $HOME /xxxx
ln -s /xxx/$(basename $HOME) $HOME
```

**Review by mithro — COMMENTED** [review]
Should we just make this part of the Kokoro image itself?

File: .github/kokoro/steps/movehome.sh (hunk 1 above)
Comment (litghost) [reply .github/kokoro/steps/movehome.sh:11]: Because that would affect the state of the VM for the next job.  That is a bad idea.

Comment (mithro) [reply .github/kokoro/steps/movehome.sh:11]: If I understand correctly the machines are throw away after they are used.

Comment (litghost) [reply .github/kokoro/steps/movehome.sh:11]: What do you see as the advantage of `mv` over `cp`.  Given that these locations are on different disks, there is no speed advantage to `mv`, and `mv` may glitch if something reloads `~`/`HOME`.  I think the `cp` is the better option.

Comment (mithro) [reply .github/kokoro/steps/movehome.sh:11]: With the cp you have to do a rm and then ln afterwards?

Comment (litghost) [reply .github/kokoro/steps/movehome.sh:11]: The `rm` and `ln` is not required at all?

**Review by mithro — APPROVED** [review]

File: .github/kokoro/steps/movehome.sh (hunk 1 above)
Comment (mithro) [reply .github/kokoro/steps/movehome.sh:11]: Was worried about things that might not be using `$HOME` for home resolution.

**Pull request merged**